core/src/main/scala/kafka/server/FinalizedFeatureChangeListener.scala

```diff
@@ -153,10 +153,12 @@ class FinalizedFeatureChangeListener(private val finalizedFeatureCache: Finalize
           // safe to ignore the exception if the thread is being shutdown. We raise the exception
           // here again, because, it is ignored by ShutdownableThread if it is shutting down.
           throw ie
-        case e: Exception => {
-          error("Failed to process feature ZK node change event. The broker will eventually exit.", e)
+        case cacheUpdateException: FeatureCacheUpdateException =>
+          error("Failed to process feature ZK node change event. The broker will eventually exit.", cacheUpdateException)
           throw new FatalExitError(1)
-        }
+        case e: Exception =>
```

Comment:
Is there a specific exception for the zk node change even processing?

Reply:
I think this is for "the rest".  e.g. session expiration